renderdoc/driver/gl/gl_resources.cpp

```diff
@@ -543,7 +547,7 @@ GLenum GetSizedFormat(const GLHookSet &gl, GLenum target, GLenum internalFormat,
   }
 
   GLint red, depth, stencil;
-  if(HasExt[ARB_internalformat_query2] && gl.glGetInternalformativ)
+  if(gl.glGetInternalformativ)
```

Comment:
What is the reason for removing this extension check? It's present to make sure that we have this extension available before calling the function, because on some implementations like mesa the function pointers may be valid even for unsupported extensions but they just do nothing and print an error message when called.